cgatcore/table.py

```diff
@@ -479,135 +479,135 @@ def main(argv=None):
         invert_match=False,
     )
 
-    (options, args) = E.start(parser, add_pipe_options=True)
+    args = E.start(parser, add_pipe_options=True)
 
-    options.parameters = options.parameters.split(",")
+    args.parameters = args.parameters.split(",")
 
-    if options.group_column:
-        options.group = True
-        options.group_column -= 1
+    if args.group_column:
+        args.group = True
+        args.group_column -= 1
 
     ######################################################################
     ######################################################################
     ######################################################################
     # if only to remove header, do this quickly
-    if options.methods == ["remove-header"]:
+    if args.methods == ["remove-header"]:
 
         first = True
-        for line in options.stdin:
+        for line in args.stdin:
             if line[0] == "#":
                 continue
             if first:
                 first = False
                 continue
-            options.stdout.write(line)
+            args.stdout.write(line)
 
-    elif options.transpose or "transpose" in options.methods:
+    elif args.transpose or "transpose" in args.methods:
 
-        read_and_transpose_table(options.stdin, options)
+        read_and_transpose_table(args.stdin, args)
```

Comment:
the read_and_transpose I think may be wrong...

Reply:
I think you need to pass unknowns=True to E.start() and any stdin arguments unknown.stdin